deploy/src/main/deploy/helm/templates/hono-service-device-connection/hono-service-device-connection-deployment.yaml

```diff
@@ -0,0 +1,68 @@
+{{- if .Values.deviceConnectionService.enabled }}
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  {{- $args := dict "dot" . "component" "service-device-connection" "name" "service-device-connection" }}
+  {{- include "hono.metadata" $args | nindent 2 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "hono.matchLabels" $args | nindent 6 }}
+  template:
+    metadata:
+      {{- include "hono.metadata" $args | nindent 6 }}
+    spec:
+      containers:
```

Comment:
How about adding tracing support by including
````
{{- include "hono.jaeger.agent" . | indent 6 }}
````
here and 
````
{{- include "hono.jaeger.clientConf" $args | indent 8 }}
````
below (as part of the `env`)  block. 
This also prevents the warning on startup:
````
12:35:42.192 [main] WARN  i.o.c.tracerresolver.TracerResolver - Error getting tracer using io.jaegertracing.tracerresolver.internal.JaegerTracerFactory@1eb6749b: Service name must not be null or empty
java.lang.IllegalArgumentException: Service name must not be null or empty
  at io.jaegertracing.internal.JaegerTracer$Builder.checkValidServiceName(JaegerTracer.java:653)
````

Reply:
absolutely, will do